code/src/components/NewTweet.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react/self-closing-comp */
+import React, { useState } from 'react';
+
+const NewTweet = ({ onTweetSubmitted }) => {
+  const [newTweet, setNewTweet] = useState('')
+
+  const handleFormSubmit = (e) => {
+    e.preventDefault()
+
+    const message = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ message: newTweet })
+    }
+
+    fetch('https://happy-thoughts-technigo.herokuapp.com/thoughts', message)
+      .then((res) => {
+        res.json()
+          .then((createdTweet) => onTweetSubmitted(createdTweet))
+          .then(() => setNewTweet(''))
+      })
+  }
+
+  const handleOnNewTweet = (e) => {
+    setNewTweet(e.target.value)
+  }
+
+  return (
+    <section className="form-container">
+      <form className="form" onSubmit={handleFormSubmit}>
+        <h1>What is making you happy right now?</h1>
+        <label htmlFor="new-thought">
+          <textarea
```

Comment:
When I view your app in the browser (chrome on mac) the  text area is sticks out of its parent container
<img width="368" alt="Skärmavbild 2022-10-10 kl  17 19 51" src="https://user-images.githubusercontent.com/105588315/194900386-554ebbdd-9a4b-4ac5-aef5-88a313cf6485.png">